### ISSUE TITLE
`label-break-value` feature used in `0.2.0` was stabilized for `1.65`, current stable version is `1.63`

### ISSUE THREAD

**Issue by zRedShift** [kelpsyberry/proc-bitfield#1]
Hi!

Saw your thread on reddit and immediately checked it out, since I'm working on an I²C driver that has a lot of register-level bitfields, and being dissatisfied with the `bitfield` crate, was just what I needed.

I started with `0.1.1` which works fine, but when I tried upgrading to `0.2.0`, the proc macro failed to compile since it uses a recently stabilized feature, `label-break-value`. It wouldn't show on `cargo check` etc. if you're working on a recent nightly.

```
error[E0658]: labels on blocks are unstable
   --> /home/ronen/.cargo/registry/src/github.com-1ecc6299db9ec823/proc-bitfield-macros-0.2.0/src/lib.rs:158:21
    |
158 |                     'parse: {
    |                     ^^^^^^
    |
    = note: see issue #48594 <https://github.com/rust-lang/rust/issues/48594> for more information
```

While the driver I'm writing will be mostly used on nightly, I want to support stable, so I'll be sticking to `0.1.1` for now, until `1.65` is out on November 3rd.

Just an FYI!

**Comment by ghost** [comment]
Fixed in c404f3808bb57bb6dfe335cdbfcdc6251cbeb041, thank you for the report!

**Comment by zRedShift** [comment]
@Kelpsy FWIW, now that 1.65 has been out for a while, it's probably safe to go back to using this feature. BTW, any plans to carve out a release with the latest commit, the one with the `#[const_trait]` annotation?

**Comment by ghost** [comment]
Yes! I think I'll still avoid using `label_break_value` just in case as it's a really minor style change anyway and might inconvenience some people on older versions, but regarding everything else, though I was wary of publishing a new release for a nightly-only fix (I... probably should have), I'll make a new one with the attribute and a few more optional trait impls (`Deref` and `From` both ways) as soon as I can (sorry, I've been quite overloaded with life stuff recently).

**Comment by zRedShift** [comment]
Understandable, no rush, and thank you!

**Comment by ghost** [comment]
I've published the fix along with the other 0.2.2 changes now!

**Comment by zRedShift** [comment]
Thank you, will look into it soon-ish!
